api/rds.proto

```diff
@@ -101,6 +101,24 @@ message RouteMatch {
   repeated HeaderMatcher headers = 6;
 }
 
+message CorsPolicy {
+  // Specifies the origins that will be allowed to do CORS requests.
+  // Can be "*" to match any origin or a list of space separated origins.
+  string allow_origin = 1;
```

Comment:
Should this be a repeated field of strings? With empty being wildcard? Are origins just domains or is there some more structure?

Reply:
I followed https://github.com/lyft/envoy/issues/300 for this. 
For the current implementation I would imagine something like `"hostA"` or `"hostA hostB"`.
Personally I would prefer a list. Empty list meaning "*" is easy to implement. Although I think it's not intuitive from a users point of view, it might stop people from doing things like `["hostA", "*"]` which makes no sense. Best would be to support regex testing, although I have no idea yet about performance implications.

Reply:
If list makes more sense just go for a list.